packages/cli/src/commands/generate/deploy/providers/aws_serverless.js

```diff
@@ -0,0 +1,98 @@
+import path from 'path'
+import fs from 'fs'
+
+import { getPaths } from 'src/lib'
+
+export const apiProxyPath = '/api'
+
+const SERVERLESS_YML = `# See the full yml reference at https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/
+service: app
+
+# Uncomment org and app if you want to integrate your deployment with the serverless dashboard. See https://www.serverless.com/framework/docs/dashboard/ for more details.
+# org: your-org
+# app: your-app
+
+plugins:
+  - serverless-dotenv-plugin
+
+custom:
+  dotenv:
+    include:
+      - # List the environment variables you want to include from your .env file here.
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  region: us-east-2 # This is the AWS region where the service will be deployed.
+  httpApi: # HTTP API is used by default. To learn about the available options in API Gateway, see https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-vs-rest.html
+    cors: true
+    payload: '1.0'
+  stackTags: # Add CloudFormation stack tags here
+    source: serverless
+    name: Redwood Lambda API with HTTP API Gateway
+  tags: # Add service wide tags here
+    name: Redwood Lambda API with HTTP API Gateway
+
+package:
+  individually: true
+
+functions:
+  ${fs
+    .readdirSync(path.resolve(getPaths().api.functions))
+    .map((file) => {
+      const basename = path.basename(file, '.js')
+      return `${basename}:
+    description: ${basename} function deployed on AWS Lambda
+    package:
+      artifact: dist/zipball/${basename}.zip # This is the default location of the zip file generated during the deploy command.
+    memorySize: 1024 # mb
+    timeout: 25 # seconds (max: 29)
+    tags: # Tags for this specific lambda function
+      endpoint: ${apiProxyPath}/${basename}
+    # Uncomment this section to add environment variables either from the serverless dotenv plugin or using serverless params
+    # environment:
+    #   YOUR_FIRST_ENV_VARIABLE: \${env:YOUR_FIRST_ENV_VARIABLE}
+    handler: ${basename}.handler
+    events:
+      - httpApi:
+          path: ${apiProxyPath}/${basename}
+          method: GET
+      - httpApi:
+          path: ${apiProxyPath}/${basename}
+          method: POST
+`
+    })
+    .join('  ')}
+`
+
+export const preRequisites = [
+  {
+    command: ['which', ['serverless']],
+    errorMessage: [
+      'Looks like serverless is not installed.',
+      'Please follow the steps at https://www.serverless.com/framework/docs/providers/aws/guide/installation/ to install serverless.',
+    ],
+  },
+]
+
+export const apiPackages = [
+  '@netlify/zip-it-and-ship-it',
+  'serverless-dotenv-plugin',
+]
+
+export const files = [
+  {
+    path: path.join(getPaths().base, 'api', 'serverless.yml'),
```

Comment:
```suggestion
    path: path.join(getPaths().base, 'serverless.yml'),
```
Even though we're only deploying the "api side" at the moment I can definitely see a future where we deploy the static side as well.